Lib/test/test_unittest/test_async_case.py

```diff
@@ -211,6 +262,9 @@ async def asyncTearDown(self):
                 events.append('asyncTearDown')
                 raise MyException()
 
+            def tearDown(self):  # must not be called
```

Comment:
This one is questionable. Right now if `asyncTearDown` fails, `tearDown` is not called.

But, following the same logic: `setUp` was successful. Maybe it should be called?

Reply:
I think that it should be called.

Also, add `tearDown` in `test_exception_in_setup`. It should be called.